sample.mvn.parent/sample.mvn.feature/pom.xml

```diff
@@ -26,28 +30,28 @@
 		</dependency>
 		<dependency>
 			<groupId>at.bestsolution.openjfx</groupId>
-			<version>11.0.0</version>
 			<artifactId>openjfx.base</artifactId>
+			<version>11.0.0</version>
```

Comment:
should use a version property here as well.

Reply:
yes, I missed that...